index.css

```diff
@@ -25,424 +17,1060 @@ a {
-  margin-right: 27px;
-  margin-top: -10px;
-  margin-bottom: 27px;
-}
+  .print-data button:hover {
+    background-color: white;
+    color: black;
+    border: solid;
+  }
 
-.about-me button:hover {
-  background-color: white;
-  color: black;
-  cursor: pointer;
-}
+  .about-me {
+    background-color: rgba(28, 26, 25, 1);
+    margin-top: 4%;
+    width: 375px;
+  }
 
-.about-me hr {
-  margin-left: 27px;
-  margin-right: 27px;
-  margin-bottom: 30px;
-}
+  .background-holder {
+    display: flex;
+    flex-direction: column;
+    background-image: url("./logos/About Me.png");
+    background-repeat: no-repeat;
+    background-size: 200px 300px;
+    background-position: right bottom;
+  }
 
-.lists {
-  display: flex;
-  flex-direction: column;
-  padding-bottom: 20px;
-}
+  .about-me h2 {
+    color: rgba(255, 107, 0, 1);
+    font-size: 72px;
+    font-weight: 400;
+    font-family: 'Crete Round', serif;
+    text-align: center;
+    padding-top: 35px;
+    margin-left: 8px;
+    margin-right: 8px;
+  }
 
-.lists li {
-  text-align: center;
-  align-items: center;
-  font-family: 'Inter', sans-serif;
-  list-style: none;
-  text-decoration: none;
-  margin-bottom: 30px;
-}
+  .about-me p {
+    font-family: 'Inter', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    margin-left: 27px;
+    margin-right: 27px;
+    margin-top: -10px;
+    margin-bottom: 27px;
+  }
+
+  .about-me button:hover {
+    background-color: white;
+    color: black;
+    cursor: pointer;
+  }
+
+  .about-me hr {
+    margin-left: 27px;
+    margin-right: 27px;
+    margin-bottom: 30px;
+  }
+
+  .lists {
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 20px;
+  }
+
+  .lists li {
+    text-align: center;
+    align-items: center;
+    font-family: 'Inter', sans-serif;
+    list-style: none;
+    text-decoration: none;
+    margin-bottom: 30px;
+  }
+
+  .lists h3 {
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    text-align: center;
+  }
+
+  .attach-icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .interesting {
+    width: 375px;
+  }
+
+  .interest-text {
+    font-family: 'Inter', sans-serif;
+    text-align: center;
+    margin-top: 80px;
+    font-size: 32px;
+    font-weight: 700;
+    margin-bottom: 60px;
+    margin-left: 10px auto;
+    margin-right: 50px auto;
+  }
+
+  .interesting input,
+  textarea {
+    font-family: 'Inter', sans-serif;
+    width: 375px;
+  }
+
+  .interesting input {
+    font-size: 15px;
+    border: none;
+    margin-bottom: 2px;
+  }
+
+  .end-name {
+    margin-bottom: 50px;
+    border-color: rgba(219, 216, 215, 0.904);
+    margin-left: 0;
+  }
+
+  .end-address {
+    margin-bottom: 50px;
+    border-color: rgba(219, 216, 215, 0.904);
+    margin-left: 0;
+  }
+
+  .interesting .text-area {
+    font-size: 15px;
+    font-style: italic;
+    background-color: #fbf8f7;
+    border: none;
+    margin-bottom: 5px;
+  }
```

Comment:
You have done well. 👍  However, you can add padding at the top &  the left side to make the text area better.
[Screenshot](https://drive.google.com/file/d/1FZsl05hAOoyTW-hOPquonA0PDSVyjpv7/view?usp=sharing)